### DISCUSSION TITLE
Help: My Copilot is no longer working

### DISCUSSION THREAD

**Discussion by AltFreq07** [community/community#40037] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Unfortunatly my Copilot has stopped working.
I believe its due to a bug with subscriptions, I signed up for the free trial, realized I could get it for free via student, so I canceled it and got the student access. After a month or so it is no longer working.

Github Billing Page
![image](https://user-images.githubusercontent.com/36876748/203889703-2c7d1251-7b65-488a-a38d-5472be367612.png)

Copilot Settings Page
![image](https://user-images.githubusercontent.com/36876748/203889575-e221dfbd-afb0-464b-bb7a-47fbbe4bcaec.png)

No settings on VS Code have changed since it stopped working.

![image](https://user-images.githubusercontent.com/36876748/203889902-cd3ac0c6-2570-48f9-a898-89e77a72c3d8.png)

Anyone got any ideas, I can't seem to remove copilot from my GitHub and readd as there is no option as seen in the Copilot settings image.

**Comment by AltFreq07** [comment]
similar issues occuring here for the technical preview https://github.com/orgs/community/discussions/35941

**Comment by 25pinjo15** [comment]
lots of people are reporting the same issues . same probleme here

**Comment by RyanZurrin** [comment]
similar issue for me, except I have a paid subscription for it, and it is failing to start for me.

**Reply by geekgao** [reply]
me too

**Reply by RyanZurrin** [reply]
[INFO] [auth] [2022-11-25T03:01:41.449Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-25T03:01:41.450Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[INFO] [auth] [2022-11-25T03:04:28.973Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-25T03:04:28.973Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[INFO] [auth] [2022-11-25T03:05:04.755Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-25T03:05:04.755Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Comment by geekgao** [comment]
I think that's a bug. Please fix it. ASAP.
I've tried uninstall then install Copilot. The same problem.

**Reply by weihanhuang-1196** [reply]
the same with me , i try to logout my account and re sign, the same

**Comment by davecheney** [comment, marked as answer]
Hello. I'm sorry you had difficulty accessing Copilot, please accept my apologies on behalf of the Copilot team. This was caused by a bad deploy which affected a small percentage of Copilot users. The change has been rolled back and you should be able to use Copilot again.

**Reply by TimHsue** [reply]
Hi, this issue still seems to be there.
On my copilot page it shows "Your subscription has ended", while I have the GitHub Student Development Kit.

**Reply by AltFreq07** [reply]
My issue has been resolved, thanks for the explanation

**Reply by darilrt** [reply]
I keep getting "No access to GitHub Copilot found" error message

**Comment by RMANOV** [comment]
The bug is not fixed - the problem continues  - not only GitHub Copilot, GitHub Copilot Nightly - too!

**Comment by haifengkao** [comment]
It still shows "your subscription has ended."
logout & login not working :(
